dimod/reference/composites/higherordercomposites.py

```diff
@@ -409,18 +411,18 @@ def sample_poly(self, poly, scalar=None, bias_range=1, poly_range=None,
 
 
 class PolyTruncateComposite(ComposedPolySampler):
-    """Composite to truncate the returned samples
+    """Composite that truncates returned samples.
 
     Post-processing is expensive and sometimes one might want to only
-    treat the lowest energy samples. This composite layer allows one to
-    pre-select the samples within a multi-composite pipeline
+    treat the lowest-energy samples. This composite layer allows one to
+    pre-select the samples within a multi-composite pipeline.
 
     Args:
         child_sampler (:obj:`dimod.PolySampler`):
             A dimod binary polynomial sampler.
 
         n (int):
-            Maximum number of rows in the returned sample set.
+            Maximum number of rows in the returned SampleSet.
```

Comment:
"sample set" is to "binary quadratic model" as "SampleSet" is to "BinaryQuadraticModel"

Reply:
Fair enough. I was on a rampage through these because I was finding "sampleSet".